backend/tracim_backend/applications/upload_permissions/lib.py

```diff
@@ -112,8 +112,9 @@ def add_permission_to_workspace(
             except SMTPRecipientsRefused:
                 logger.warning(
                     self,
-                    "Upload Permission initied by {email} but SMTP server refuse to send notification".format(
-                        email=self._user.email
+                    "Upload Permission initied by user {} but SMTP "
+                    "server refuse to send notification".format(
+                        self._user.username or self._user.email
```

Comment:
so we decide to show username first here ?

Reply:
see cfb7110

Reply:
ok.